pkg/packageinstall/app.go

```diff
@@ -53,7 +60,11 @@ func NewApp(existingApp *v1alpha1.App, pkgInstall *pkgingv1alpha1.PackageInstall
 	desiredApp.Spec = *pkgVersion.Spec.Template.Spec
 	desiredApp.Spec.ServiceAccountName = pkgInstall.Spec.ServiceAccountName
 	if pkgInstall.Spec.SyncPeriod == nil {
-		desiredApp.Spec.SyncPeriod = &metav1.Duration{Duration: time.Minute * 10}
+		if opts.DefaultSyncPeriod != 0 {
+			desiredApp.Spec.SyncPeriod = &metav1.Duration{Duration: opts.DefaultSyncPeriod}
+		} else {
+			desiredApp.Spec.SyncPeriod = DefaultSyncPeriod
+		}
```

Comment:
Since we are doing this in config now:
```go
	const defaultSyncPeriod = 10 * time.Minute
	if gc.data.packageInstallDefaultSyncPeriod != 0 {
		return gc.data.packageInstallDefaultSyncPeriod
	}
	return defaultSyncPeriod
```
We can just do this:
```suggestion
	        desiredApp.Spec.SyncPeriod = &metav1.Duration{Duration: opts.DefaultSyncPeriod}
```

Reply:
Good point, i tried this however it results into `syncPeriod: 0` now using the default that is coming from `config`